Transfer/Sources/TransferDestinationCell.swift

```diff
@@ -112,18 +112,25 @@ final class TransferDestinationCell: GenericCell<HyperwalletTransferMethod> {
 extension TransferDestinationCell {
     func configure(transferMethod: HyperwalletTransferMethod) {
         titleLabel.text = transferMethod.title
+        titleLabel.font = Theme.Label.captionOne
```

Comment:
This is being set from ThemeManager so remove it from here. As I have said so many times earlier, these values are set through ThemeManager and not directly.

Reply:
Removed